cloudapi/config.go

```diff
@@ -28,6 +28,7 @@ type Config struct {
 	TestRunDetails null.String `json:"testRunDetails" envconfig:"K6_CLOUD_TEST_RUN_DETAILS"`
 	NoCompress     null.Bool   `json:"noCompress" envconfig:"K6_CLOUD_NO_COMPRESS"`
 	StopOnError    null.Bool   `json:"stopOnError" envconfig:"K6_CLOUD_STOP_ON_ERROR"`
+	UseVersion2    null.Bool   `json:"useVersion2" envconfig:"K6_CLOUD_USE_VERSION_2"`
```

Comment:
Maybe this can be something a bit more flexible?

```suggestion
	UsedAPIVersion    null.Int   `json:"usedVersion" envconfig:"K6_CLOUD_USED_VERSION"`
```

And, if the used version is higher than the supported one, k6 can exit with an error? :thinking:

Reply:
Addressed in https://github.com/grafana/k6/pull/2963/commits/ea2c1ef47959ab3ae81474440aa89009a34f86ae